core/main/src/state/cap/generic_cap_state.rs

```diff
@@ -49,6 +49,7 @@ impl GenericCapState {
             FireboltCap::Short("token:platform".to_owned()),
             FireboltCap::Short("usergrant:acknowledgechallenge".to_owned()),
             FireboltCap::Short("usergrant:pinchallenge".to_owned()),
+            // <pca> Do we need to add "discovery:interest" etc. here or just in the manifests? </pca>
```

Comment:
I'm not sure why these are hardcoded here, I think when `GenericCapState::new()` is called the manifest has already been ingested so what's the point, why not just include these in the manifest?

Reply:
When Ripple starts up these capabilities are set to unavailable. So any calls for these providers would fail with unavailable error. 

Once the providers are registered the broker would then set these capabilities as available. From that point the consecutive calls would pass

Reply:
@satlead: Right but these get set as a result of `BootstrapState::build()` being called, which is before the providers are registered. Because of that they might as well just be defined in the manifest right?